src/Http/Headers/src/MediaTypeHeaderValue.cs

```diff
@@ -645,7 +646,9 @@ private static int GetMediaTypeExpressionLength(StringSegment input, int startIn
             }
             else
             {
-                mediaType = input.Substring(startIndex, typeLength) + ForwardSlashCharacter + input.Substring(current, subtypeLength);
+                var forwardSlashCharacter = ForwardSlashCharacter;
+                var forwardSlash = MemoryMarshal.CreateReadOnlySpan(ref forwardSlashCharacter, 1);
```

Comment:
This is an unsafe equivalent. Prefer instead:

```cs
ReadOnlySpan<char> forwardSlash = stackalloc char[] { '/' };
```

Reply:
You don't even need that... just pass `"/"` as the argument to Concat below.

Reply:
I wish we had gifs on github. I have the perfect one for this sequence...

Reply:
Something like...
![](https://media.tenor.com/images/ed92b998e634f3a38d5d061a4157d5ce/tenor.gif)

Reply:
I find myself being very curious what yours was going to be... bring on the gifs :)